views/show_wall.erb

```diff
@@ -0,0 +1,13 @@
+<h2><%= @wall.title %>, by <%= @wall.created_by %></h2>
+<p class="description"><%= @wall.description %></p>
+
+<form action="/walls/<%= @wall.id %>/edit" method= "get">
```

Comment:
When you just need to send a `GET` request, you don't necessarily need a form. You can just use a link (an `<a>` tag). If you want to style the `<a>` tag, you still can:

```html
<a class="btn btn-primary">...</a>
```